src/sql/src/error.rs

```diff
@@ -149,6 +149,14 @@ pub enum Error {
         location: Location,
     },
 
+    #[snafu(display("Unrecognized database option key: {}, value: {}", key, value))]
+    InvalidDatabaseOptionValue {
+        key: Ident,
+        value: u64,
+        #[snafu(implicit)]
+        location: Location,
+    },
```

Comment:
Could we reuse the `InvalidSql` variant? Since `limit` isn't a database option now. The error message might be `limit xxx is not supported`